### DISCUSSION TITLE
Using token from a volume for ARC.

### DISCUSSION THREAD

**Discussion by adithya-intuit** [actions/actions-runner-controller#2304] [Questions]
Hey all,

I have my PAT stored in a volume, is there any way I can use this to authenticate? By default it uses token from the environment variable. I also see that the final container image is super bare, so I cannot use the file and modify the entrypoint.

Thanks in advance!

**Comment by jslay88** [comment]
I, too, would love to know how to use file based secrets for the PAT, as the production clusters we use, do not allow tokens as environment variables, or to use Kubernetes secrets. All of our secrets are brought in through ASCP
